### PR TITLE
Prisma Access - Added tunnel health playbook

### PR DESCRIPTION
**Pull request by liorkol**
## Status
- [x] Ready
## Description
Playbook to test Prisma Access connection health. Relies on existing integration commands.

## Screenshots
Prisma_Access_-_Connection_Health_Check

## Minimum version of Demisto
- [x] 5.5.0

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

File: Packs/PrismaAccess/Playbooks/playbook-Prisma_Access_-_Connection_Health_Check.yml [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+      type: regular
+      iscommand: true
+      brand: SlackV2
+    nexttasks:
+      '#none#':
+      - "9"
+    scriptarguments:
+      blocks: {}
+      channel:
+        simple: netops
```
Comment (michalgold) [inline Packs/PrismaAccess/Playbooks/playbook-Prisma_Access_-_Connection_Health_Check.yml:164]: A customer might not have "netops" channel and right now it is hardcoded, which will cause the playbpook to fail for the majority of customers. We can add this as playbook input instead. 

Also, what happens if the customer does not even use slack? Slack should be optional. Add a check if integration is enabled and a channel was provided as playbook input.

File: Packs/PrismaAccess/Playbooks/playbook-Prisma_Access_-_Connection_Health_Check.yml [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+    skipunavailable: false
+    quietmode: 0
+  "9":
+    id: "9"
+    taskid: 8bea80c0-59c0-487f-8a6f-05b74f4df71b
+    type: regular
+    task:
+      id: 8bea80c0-59c0-487f-8a6f-05b74f4df71b
+      version: -1
+      name: Remediate tunnel issue
```
Comment (michalgold) [inline Packs/PrismaAccess/Playbooks/playbook-Prisma_Access_-_Connection_Health_Check.yml:342]: Adding a generic recommendation (what actions will be taken) in the description is beneficial.

File: Packs/PrismaAccess/Playbooks/playbook-Prisma_Access_-_Connection_Health_Check.yml (hunk 1 above)
Comment (liorkol) [reply Packs/PrismaAccess/Playbooks/playbook-Prisma_Access_-_Connection_Health_Check.yml:164]: @michalgold Removing Slack step

File: Packs/PrismaAccess/Playbooks/playbook-Prisma_Access_-_Connection_Health_Check.yml (hunk 2 above)
Comment (liorkol) [reply Packs/PrismaAccess/Playbooks/playbook-Prisma_Access_-_Connection_Health_Check.yml:342]: Adding description.

**Comment by yaron-libman** [comment]
reviewed

**Pull request merged**